ssl/statem/extensions.c

```diff
@@ -121,6 +121,14 @@ static const EXTENSION_DEFINITION ext_defs[] = {
         final_renegotiate
     },
     {
+        TLSEXT_TYPE_max_fragment_length,
+        SSL_EXT_CLIENT_HELLO | SSL_EXT_TLS1_3_ENCRYPTED_EXTENSIONS
+        | SSL_EXT_TLS1_2_SERVER_HELLO | SSL_EXT_TLS1_3_SERVER_HELLO,
```

Comment:
Not quite. TLS 1.3 draft 19 says that max_fragment_length should appear in EE not SH (and definitely not both as you have it). So it should be:
SSL_EXT_CLIENT_HELLO | SSL_EXT_TLS1_2_SERVER_HELLO | SSL_EXT_TLS1_3_ENCRYPTED_EXTENSIONS